recipe/meta.yaml

```
@@ -34,95 +45,147 @@ build:
         - cctools                # [osx]
-        - compiler-rt            # [osx]
+        - llvmdev {{ bootstrap_version }}  # [osx]
         - llvm-tools             # [osx]
-        - {{ compiler('cxx') }}  # [not osx]
+        - sysroot_{{ target_platform }}  # [linux]
+        - {{ compiler('cxx') }}   # [not linux]
+        - {{ compiler('c') }}     # [not linux]
       files:
         - test_sources
         - compile_test.sh   # [unix]
         - compile_test.bat  # [win]
       commands:
         # Dummy var for different hashes
         - echo {{ dummy_var }}                                    # [osx]
+        - echo {{ sys_abi }}                                      # [osx]
         - if not exist %LIBRARY_INC%\\c++\\v1\\iterator exit 1    # [win]
         - test -f $PREFIX/include/c++/v1/iterator                 # [unix]
         - bash compile_test.sh                                    # [unix]
         - compile_test.bat                                        # [win]
-        # !!!!!!!!!!!! README !!!!!!!!!!!!!!!!!!
         # we do not ship libcxxabi on OSX - this breaks exception passing
         - if [ -f $PREFIX/lib/libc++abi.dylib ]; then exit 1; fi  # [osx]
         - test -f $PREFIX/lib/libc++abi.so                        # [linux]
+      {% endif %}
+
 
-      # Following 2 projects had issues with libc++, so check them
-      # before releasing just to be sure.
-      # downstreams:   # [osx]
-      #  - python-symengine  # [osx]
-      #  - openturns         # [osx]
-      #  - libcxx-testing    # [osx]
+  # Honestly not sure where the 'hardening' is used (or if) but leaving it here for
+  # consistency. There are also some comments in here that I think may be out of date
+  # but I haven't had a chance to check on them because they seem reasonable.
+  - name: libcxx
+    build:
+      string: h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}        # [hardening == "none"]
+      string: debug_h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [hardening == "debug"]
+      run_exports:                                      # [hardening == "debug"]
+        # packages built with hardened lib must not be installable without extra label
+        - libcxx =*=debug*                              # [hardening == "debug"]
+        - libcxx                                        # [hardening == "none"]
+      ignore_run_exports_from:
+        # we want to ignore libstdcxx, but not libgcc; the latter also comes in through
+        # compiler('c'), and conda keeps that despite ignoring the cxx run-exports
+        - {{ compiler('cxx') }}
+    files:
+        # I'm not sure why windows is included here, I'm guessing experimental?
+      - lib/libc++.so               # [linux]
+      - lib/libc++.so.*             # [linux]
+      - lib/libc++.dylib            # [osx]
+      - lib/libc++.*.dylib          # [osx]
+      - Library/bin/c++*.dll        # [win]
+      - lib/libc++.a                # [unix]
+      - lib/libc++experimental.*    # [unix]
```

```suggestion
      - lib/libc++.so.*                 # [linux]
      - lib/libc++experimental.so.*     # [linux]
      - lib/libc++.*.dylib              # [osx]
      - lib/libc++experimental.*.dylib  # [osx]
      - Library/bin/c++*.dll            # [win]
      
```
The rest of these should be present in the `-devel` package.